flare/output.py

```diff
@@ -355,7 +359,8 @@ def write_xyz_config(self, curr_step, structure, dft_step,
         self.write_xyz(curr_step=curr_step, pos=structure.positions,
                        species=structure.species_labels, filename='xyz',
                        header=header,
-                       forces=forces, stds=stds, forces_2=forces_2)
+                       forces=forces, stds=stds, forces_2=forces_2,
```

Comment:
I implemented a `print_as_xyz` method in the `Structure` class which is fairly flexibile; perhaps this could be used here?